doc/sphinx/source/quickstart/installation.rst

```diff
@@ -392,3 +428,159 @@ To do this, run the tool with:
+solve the environment, because now it can ignore any packages that were built
+for other Python versions.
+
+Problems with proxies
+---------------------
+If you are installing ESMValTool from source from behind a proxy that does not
+trust the usual PyPI URLs you can declare them with the option
+``--trusted-host``, e.g.
+
+.. code-block:: bash
+
+    pip install --trusted-host=pypi.python.org --trusted-host=pypi.org --trusted-host=files.pythonhosted.org -e .[develop]
+
+If R packages fail to download, you might be able to solve this by
+setting the environment variable ``http_proxy`` to the correct value, e.g.
+in bash:
+
+.. code-block:: bash
+
+    export http_proxy=http://user:pass@proxy_server:port
+
+the username and password can be omitted if they are not required. See e.g.
+`here <https://support.rstudio.com/hc/en-us/articles/200488488-Configuring-R-to-Use-an-HTTP-or-HTTPS-Proxy>`__
+for more information.
+
+Installation of R packages fails
+--------------------------------
+Problems have been reported if the ``R`` interpreter was made available
+through the ``module load`` command in addition to installation from conda.
+ESMValTool works fine with either way of installing R, but do not try to use
+both installation methods at the same time.
+If your ESMValTool conda environment is called ``esmvaltool`` and you want to
+use the R interpreter installed from conda, the path to the R interpreter should
+end with ``conda/envs/esmvaltool/bin/R``.
+When the conda environment for ESMValTool is activated, you can check which R
+interpreter is used by running
+
+.. code-block:: bash
+
+    which R
+
+The Modules package is often used by system administrators to make software
+available to users of scientific compute clusters.
+To list any currently loaded modules run ``module list``, run ``module help``
+or ``man module`` for more information about the Modules package.
+
+Problems when using ssh
+-----------------------
+If you log in to a cluster or other device via SSH and your origin
+machine sends the ``locale`` environment via the SSH connection,
+make sure the environment is set correctly, specifically ``LANG`` and
+``LC_ALL`` are set correctly (for GB English UTF-8 encoding these
+variables must be set to ``en_GB.UTF-8``; you can set them by adding
+``export LANG=en_GB.UTF-8`` and ``export LC_ALL=en_GB.UTF-8``) in your
+origin or login machines’ ``.profile``.
+
+Problems when updating the conda environment
+--------------------------------------------
+Usually conda is much better at solving new environments than updating older
+environments, so it is often a good idea to create a new environment if updating
+does not work. See also `Conda fails to solve the environment`_.
+
+Do not run ``conda update --update-all`` in the ``esmvaltool``
+environment since that will update some packages that are pinned to
+specific versions for the correct functionality of the tool.
```

Comment:
nice!! very nicely summarized and I really like the possible problems (Troubleshooting) bit, was this in before? I think we should add the vase when conda is unable to retrieve a certain package because of a 500/501/504 issue (network or remote server) - this is more and more frequent these days for some reason, and a retry usually solves the issue. On a different topic, I can't see the bit about Julia no more - have I missed it or is it missing? I think it's important we explicitly say how to symlink the executable to the envs/esmvaltool/bin location - it's not fully and user-friendly described on the Julia website :+1:

Reply:
> I think we should add the case when conda is unable to retrieve a certain package because of a 500/501/504 issue (network or remote server problem) - this is more and more frequent these days for some reason, and a retry usually solves the issue.

Sounds good, would you be willing to make a pull request to add that to the section with common problems after this one is merged?

> I can't see the bit about Julia no more - have I missed it or is it missing? I think it's important we explicitly say how to symlink the executable to the envs/esmvaltool/bin location - it's not fully and user-friendly described on the Julia website

Indeed I removed it, but we have a really nice command by command description in the tutorial: https://esmvalgroup.github.io/ESMValTool_Tutorial/02-installation/index.html and the tutorial is mentioned at the top of this document for those who would like more detail. Do you think that could work?

Reply:
point 1: absolutely! Let's merge this then I'll do that.
point 2: I think it's best we have complete instructions here, even if a much nicer and user-friendly set of instructions is on the tutorial; the documentation is usually the first point of entry and if users get stuck coz something is missing from it a good fraction of them would either ask (the UX group or us needing to spend extra time) or they simply give up :+1:

Reply:
OK, I added a lot more instructions, could you have another look please?

Reply:
doing it now :+1:

Reply:
very cool - let me open an issue about adding more troubleshooting info after this is merged